app/controllers/declaration/ItemsSummaryController.scala

```diff
@@ -60,4 +62,17 @@ class ItemsSummaryController @Inject()(
       case Some(data) => data.onwardSupplyRelief == FiscalInformation.AllowedFiscalInformationAnswers.yes
       case None       => false
     }
+
+  def addItem(): Action[AnyContent] = (authenticate andThen journeyType) { implicit request =>
+    sessionId.map(updateCache(_, ExportItem()))
```

Comment:
Similar to other comments, the `Option[Future]`  is never used
```
sessionId.match {
  case Some(id) => updateCache(id, ExportItem()))
  case None  => Future.successful()
} map (_ =>  Redirect(...))
```

Reply:
Also with this one, technically the redirect and subsequent page load could happen before the cache update future completes. So I could fill in the form, click add, and the page could load without the updated content

Reply:
we have addressed this